content/posts/newsletter-020/index.md

```diff
@@ -868,6 +868,27 @@ can get glimpses of its development by following the author on [Twitter][h3r2tic
 [@h3r2tic]: https://github.com/h3r2tic
 [h3r2tic-twitter]: https://twitter.com/h3r2tic
 
+### building-blocks v0.6.0
+
+![LOD Terrain](https://media.githubusercontent.com/media/bonsairobo/building-blocks/main/examples/screenshots/lod_terrain.png)
```

Comment:
Ideally this should be replaced with a local copy of the image (to avoid broken links if your repo moves, for example) - would it be possible for you to do that? Let us know if not, we can help.